test/etc/UnstableTests.txt

```diff
@@ -38,4 +38,8 @@ WMCore_t.WorkQueue_t.DataStructs_t.WorkQueueElement_t.WorkQueueElementTest:testP
 WMCore_t.WMBS_t.JobSplitting_t.FileBased_t.FileBasedTest:testLocationSplit
 WMCore_t.Services_t.LogDB_t.LogDB_t.LogDBTest:test_heartbeat
 WMCore_t.WorkQueue_t.DataStructs_t.WorkQueueElement_t.WorkQueueElementTest:testPossibleSitesLocationFlags
-WMCore_t.WorkQueue_t.DataStructs_t.WorkQueueElement_t.WorkQueueElementTest:testPossibleSites
\ No newline at end of file
+WMCore_t.WorkQueue_t.DataStructs_t.WorkQueueElement_t.WorkQueueElementTest:testPossibleSites
+WMCore_t.Configuration_t.ConfigurationTest:testF
```

Comment:
```suggestion
-WMCore_t.Configuration_t.ConfigurationTest:testF
```

I would remove this line, I saw and fixed this while working on #10575. For the time being, the fix is in #10575, but we can move it to a separate PR

Reply:
I can take it out from this PR then. New change coming in in a minute.

Reply:
perfect, thanks!